ssg/templates.py

```diff
@@ -360,7 +363,7 @@ def build_all_rules(self):
                 continue
             langs_to_generate = self.get_langs_to_generate(rule)
             self.build_rule(
-                rule.id_, rule.title, rule.template, langs_to_generate, platforms=rule.platforms)
+                rule.id_, rule.title, rule.template, langs_to_generate, rule, platforms=rule.platforms)
```

Comment:
When I see it I can see that there are these parameters passed: `rule.id_`, `rule.title`, `rule.template`, which are all attributes of the `rule` object and at the same time the whole `rule` object is passed. I feels superflous. I would rename the new parameter of the `build_rule` to `identifiers`, changed the line 307 accordingly and here I would pass `rule.identifies` as an argument.